pylift/generate_data.py

```diff
@@ -63,3 +67,93 @@ def dgp(N=1000, n_features=3, beta=[-3,-8,13,-8], error_std=0.5, tau=3, tau_std=
 
     df = pd.DataFrame(np.concatenate((X, treatment.reshape(-1,1), y.reshape(-1,1)), axis=1), columns=names)
     return df
+
+
+
+def sim_pte(N = 1000, n_features = 20, beta = None, rho = 0, sigma =  np.sqrt(2), beta_den = 4, discrete_outcome=False, seed=None):
```

Comment:
for pep8 style, you should remove the spaces around the equal signs in function definitions. but not a big deal - we can autopep8 it later.

Reply:
I'll push a fix. This is what PR and review is for.

Reply:
Fixe pushed.